packages/running/src/index.tsx

```diff
@@ -689,6 +689,7 @@ class Section extends PanelWithToolbar {
       );
     }
     this.toolbar.addClass('jp-RunningSessions-toolbar');
+    this._toolbar.addAttribute('aria-label', this.title.label + ' toolbar');
```

Comment:
We need to make this translatable with `%1` see https://jupyterlab.readthedocs.io/en/stable/extension/internationalization.html#internationalization-and-localization